sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Breaking Changes
 
+####  Dropped Support for TypeScript 4.1
+- In response to compatibility issues with Node 18, support for TypeScript version 4.1 has been discontinued. The new minimum supported TypeScript version is now 4.2. Please ensure your environment is updated accordingly.
```

Comment:
I think we should not mention about the compatability issue with Node 18, just state like this , "We have opted to discontinue support for TypeScript version 4.1. Consequently, the minimum supported TypeScript version has been elevated to 4.2. Kindly ensure that your environment is promptly updated to align with these changes."

Reply:
FWIW we recently updated our guidelines to align with DefinitelyTyped here: https://azure.github.io/azure-sdk/typescript_design.html#ts-support-ts